logaccess/loki/src/main/kotlin/LokiLogFileProvider.kt

```diff
@@ -159,10 +159,9 @@ class LokiLogFileProvider(
         source: LogSource,
         levels: Set<LogLevel>
     ): String {
-        val levelCriterion = levels.joinToString(" or ") { """level="${it.name}"""" }
-        return """{namespace="${config.namespace}",container=~"${source.name.lowercase()}.*",""" +
-                """run_id="$ortRunId"} | logfmt level | $levelCriterion """ +
-                "| drop __error__, __error_details__, level"
+        val levelCriterion = levels.joinToString("|") { it.name }
```

Comment:
Should this commit close https://github.com/eclipse-apoapsis/ort-server/issues/876?